assets/src/edit-story/masks/display.js

```diff
@@ -83,7 +84,11 @@ export default function WithMask({
     >
       <svg width={0} height={0}>
         <defs>
-          <clipPath id={maskId} clipPathUnits="objectBoundingBox">
+          <clipPath
+            id={maskId}
+            transform={'scale(1 ' + heightMultiplier + ')'}
```

Comment:
Minor nit, might read a bit better:

```suggestion
            transform={`scale(1 ${heightMultiplier})`}
```